platform-tooling-support-tests/src/test/java/platform/tooling/support/tests/JavacModulesTests.java

```diff
@@ -64,22 +63,23 @@
 			}
 		}
 
-		private Stream<Path> getMainOutputDirs() {
-			try {
-				return Files.list(javaClassesDir) //
+		private List<Path> getMainOutputDirs() {
+			try (var files = Files.list(javaClassesDir)) {
+				return files //
 						.filter(file -> Files.isDirectory(file)) //
-						.filter(dir -> dir.getFileName().toString().startsWith("main"));
+						.filter(dir -> dir.getFileName().toString().startsWith("main")) //
```

Comment:
"//" redundant here ?

Reply:
Why?

It's a playground branch/PR anyway. As long as our style checker is happy, I'm happy too.

Reply:
Yeah. Thanks.